SkillChat.Client.ViewModel/SelectMessages.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Windows.Input;
+using PropertyChanged;
+using ReactiveUI;
+using ServiceStack;
+using Splat;
+using Avalonia;
+using Avalonia.Input.Platform;
+
+using System.Collections.ObjectModel;
+using System.Threading.Tasks;
+
+namespace SkillChat.Client.ViewModel
+{
+    [AddINotifyPropertyChangedInterface]
+    public class SelectMessages
+    {
+        /// <summary>
+        /// Переменная - флаг для вкл./выкл. режима выбора сообщений
+        /// </summary>
+        public bool IsTurnedSelectMode { get; set; }
+
+        public int CountCheckedMsg { get; set; }
+
+        /// <summary>
+        /// Коллекция для временного (оперативного) хранения выбранных сообщений
+        /// </summary>
+        public ObservableCollection<MessageViewModel> SelectedMessagesTempCollection { get; set; }
```

Comment:
Название очень страшное
```suggestion
        public ObservableCollection<MessageViewModel> SelectedCollection { get; set; }
```

Reply:
Переименовал